internal/dkg/execution.go

```diff
@@ -205,6 +177,15 @@ func (d *Process) startDKGExecution(
 			return nil, result.Error
 		}
 
+		var transitionTime int64
+
+		if current.Epoch == 1 {
+			transitionTime = current.GenesisTime.Unix()
+		} else {
+			roundsUntilTransition := 10
+			currentRound := common.CurrentRound(time.Now().Unix(), current.BeaconPeriod, current.GenesisTime.Unix())
+			transitionTime = common.TimeOfRound(current.BeaconPeriod, current.GenesisTime.Unix(), currentRound+uint64(roundsUntilTransition))
```

Comment:
How is the transitionTime related to the timeout period of 24h atm, IIRC?

Is the timeout happening prior to execution and transitionTime expected to happen after execution?

Reply:
Timeout is a configurable period after which nodes should assume a DKG has failed, so TransitionTime and Timeout are independent.
Right now there isn't an automated process to transition to `TimedOut` in the DKG, though nodes will stop storing states for a given epoch successfully (and require an `Abort`)